src/main/resources/rotations.yml

```diff
@@ -0,0 +1,18 @@
+#
+# Rotations sample configuration
+#
+# Each new rotation should follow the same configuration structure, within this file
+#
+# NOTE: Map names are case-sensitive
+#
+
+default:
+  enabled: true
+  players: 1
+  position: 0                 # This helps the server resume rotations in the position they were before a server shutdown/restart
```

Comment:
How about instead of storing an index, we store the map name of the first map. So:
`next_map: Harb`

That makes it easier for users to configure this.

Reply:
Resolved in https://github.com/Electroid/PGM/pull/114/commits/8047f78e56826a1ca8430811204fda8e6f95447c